configmap/repl-ctl/start/start.sh

```diff
@@ -15,12 +15,14 @@ entrypoint --log-level DEBUG smig-update --repl-connection "{{.ReplicationDataba
 entrypoint --log-level DEBUG replication-controller \
     --db-uri "{{.ReplicationDatabaseURL}}" \
     --db-admin-uri "{{.ReplicationDatabaseRootURL}}" \
-    --qserv-czar-db="{{.CzarDatabaseRootURL}}" \
+    --qserv-czar-db="{{.CzarDatabaseRootURL}}"  \
```

Comment:
Is it just an extra space (probably not needed)?

Reply:
:+1: